app/lib/containers/hyperterm.js

```diff
@@ -42,7 +43,13 @@ class HyperTerm extends Component {
     keys.bind('command+6', moveTo.bind(this, 5));
     keys.bind('command+7', moveTo.bind(this, 6));
     keys.bind('command+8', moveTo.bind(this, 7));
-    keys.bind('command+9', moveTo.bind(this, 8));
+    keys.bind('command+9', () => {
+      const lastTerm = this.terms.getLastTerm();
+      if (lastTerm) {
```

Comment:
I'm not sure if this check is necessary - can there be a scenario where this can be reached without at least one active term being available?